test/fixtures/webpack-message-formatting/index.test.js

```diff
@@ -59,7 +59,10 @@ test('formats missing package', async () => {
     path.join(testSetup.testDirectory, 'src', 'App.js')
   );
 
-  const { stdout, stderr } = await testSetup.scripts.build();
+  let { stdout, stderr } = await testSetup.scripts.build();
```

Comment:
The snapshot includes unix-style paths, but the stderr returned here on Windows includes Windows-style paths (causing these tests to fail prior to this change). This is a somewhat hacky way to "normalize" the paths (i.e. make them unix-style) prior to comparison with the snapshot. Jest does something similar (fwiw).